crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -559,11 +559,21 @@ impl ExternalApiWireError {
 
 #[derive(Debug, serde::Serialize, serde::Deserialize, SerializedBytes, Clone)]
 /// Filter for [`AdminRequest::ListApps`].
+///
+/// App Status is a combination of two pieces of independent state:
+/// - Enabled/Disabled, which is a designation set by the user via the conductor admin interface.
```

Comment:
As apps with deferred memproofs can be enabled from the app interface too, I suggest
```suggestion
/// - Enabled/Disabled, which is a designation set by the user via the conductor interface.
```